### PR TITLE
Enable running travis tests using cron

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/640/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1469/ : FAILURE in 58s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1730/ : FAILURE in 1m 57s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/641/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1470/ : FAILURE in 1m 07s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1731/ : FAILURE in 1m 03s

**Comment by panjan** [comment]
Testing cron job: https://travis-ci.org/panjan/gooddata-ruby/builds/433601595

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/132/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1472/ : FAILURE in 1m 31s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1733/ : FAILURE in 1m 19s
- gooddata-ruby-integration-tests-zuul : SKIPPED
- gooddata-ruby-lcm-integration-tests-zuul : SKIPPED

**Comment by panjan** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/644/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1535/ : SUCCESS in 32m 17s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul/99/ : FAILURE in 19m 32s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: lib/gooddata/lcm/actions/collect_segments.rb [hunk 1]
```diff
@@ -35,7 +35,8 @@ class << self
         def call(params)
           data_product = params.data_product
           data_product_segments = data_product.segments
-          params.gdc_logger.info("Domain segments: #{data_product_segments}")
+          segment_names = data_product_segments.map(&:segment_id)
+          params.gdc_logger.debug("Domain segments: #{segment_names}")
```
Comment (yenkins) [inline lib/gooddata/lcm/actions/collect_segments.rb]: info: Calls 'params.gdc_logger' 3 times (DuplicateMethodCall) (reek)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/645/ : SUCCESS in 19s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1474/ : SUCCESS in 2m 28s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1735/ : FAILURE in 1m 28s

**Comment by panjan** [comment]
extended test

File: .travis.yml [hunk 2]
```diff
@@ -1,33 +1,34 @@
 language: ruby
 cache: bundler
 stages:
-  - unit
-  - name: integration
-    if: type = api
+- unit
```
[the remark this answers is not in the document]
Comment (panjan) [reply .travis.yml]: It's auto-formatted by the travis gem. ✔️

File: .travis.yml [hunk 3]
```diff
@@ -1,33 +1,34 @@
...
-    - stage: integration
-      script: bundle exec rake test:sdk
-      rvm: jruby-9.1.7
-    - stage: integration
-      script: bundle exec rake test:project
-      rvm: jruby-9.1.7
+  - stage: unit
+    script: bundle exec rake test:unit
+    rvm:
+    - 2.2
```
[the remark this answers is not in the document]
Comment (panjan) [reply .travis.yml]: Auto-formatted by the travis gem.

**Pull request merged**